AlignmentPalette/Alignment.glyphsPalette/Contents/Info.plist

```diff
@@ -14,25 +12,23 @@
 	<string>6.0</string>
 	<key>CFBundleName</key>
 	<string>Alignment Palette</string>
-	<key>CFBundlePackageType</key>
-	<string>BNDL</string>
 	<key>CFBundleVersion</key>
-	<string>1.11</string>
+	<string>14</string>
```

Comment:
What is the reason for not using the same as CFBundleShortVersionString?

Reply:
Behause that is how they ares supposed to be used.

Reply:
Really?
https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleshortversionstring

Reply:
Really?
https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleversion

Reply:
You are right. Last time I checked, It was a X.Y.Z in CFBundleShortVersionString and a single build number in CFBundleVersion. For me, that is the only practically way as I need to distinguish different build with the same version number (CFBundleShortVersionString).